ops/services/web_application_firewall/main.tf

```diff
@@ -129,6 +144,7 @@ resource "azurerm_web_application_firewall_policy" "sr_waf_policy" {
           "942200",
           "942260",
           "942330",
+          "942361",
```

Comment:
I'm seeing [here](https://docs.microsoft.com/en-us/azure/web-application-firewall/ag/application-gateway-crs-rulegroups-rules?tabs=owasp32) that `942361` is excluding `Detects basic SQL injection based on keyword alter or union`.

For my own curiosity, is this a commonly excluded rule or something specific to our application that we need to do this?

Reply:
Great question! This is specific to our application. In certain areas, such as a street or facility name, the word "Union" is perfectly valid. As an example, here in Chicago, we have a street called Union Avenue; if this rule were left in place, any testing facility or patient who resides on that street wouldn't be able to use the application, as the WAF would prevent requests with their information from going through.

This is one of the downsides with a WAF...default implementations tend to be pretty strict, and we can run into trouble trying to loosen things enough to run the application. That's why this process can routinely take a few weeks.

Reply:
Oh wow! I see thank you. That's a very helpful example and explanation!! 😹